pod.go

```diff
@@ -224,8 +224,9 @@ type Cmd struct {
 	Envs    []EnvVar
 	WorkDir string
 
-	User  string
-	Group string
+	User             string
+	Group            string
+	AdditionalGroups []string
```

Comment:
I am fine with a having a clear distinction between a process primary and supplementary groups as this is a POSIX definition. But then the naming should match POSIX and not OCI:

```
PrimaryGroup        string
SupplementaryGroups []string